src/test/java/org/cactoos/list/ListEnvelopeTest.java

```diff
@@ -66,13 +63,12 @@ public void returnsListIteratorWithUnsupportedSet() {
                 inner.add("three");
                 return inner;
             }
-        ) { };
-        final ListIterator<String> iterator = list.listIterator();
-        iterator.next();
+        ) {
+        };
+        final ListIterator<String> iterator = list.listIterator(1);
```

Comment:
@proshin-roman See above, about `ListIterator(List, int)` constructor; if implemented, it could be used here.